prow/cmd/status-reconciler/main.go

```diff
@@ -20,30 +20,46 @@ import (
 	"flag"
 	"os"
 	"os/signal"
+	"strings"
 	"syscall"
 
 	"github.com/sirupsen/logrus"
-	"k8s.io/test-infra/prow/pjutil"
+	"k8s.io/apimachinery/pkg/util/sets"
 
 	"k8s.io/test-infra/pkg/flagutil"
 	"k8s.io/test-infra/prow/config"
 	"k8s.io/test-infra/prow/config/secret"
 	prowflagutil "k8s.io/test-infra/prow/flagutil"
 	_ "k8s.io/test-infra/prow/hook"
 	"k8s.io/test-infra/prow/logrusutil"
+	"k8s.io/test-infra/prow/pjutil"
 	"k8s.io/test-infra/prow/plugins"
 	"k8s.io/test-infra/prow/statusreconciler"
 )
 
+type blacklistFlags struct {
+	members sets.String
```

Comment:
Consider using https://github.com/kubernetes/test-infra/blame/master/prow/flagutil/strings.go or making this a generic 'set of strings' flag and adding it to `flagutil`.

Reply:
Will do in a follow-up